src/tools/pub/cmd/pub-outdated.md

```diff
@@ -0,0 +1,236 @@
+
+
+```terminal
+$ pub outdated
+Package              Current   Upgradable  Resolvable  Latest    
+dependencies: all up-to-date
+
+dev_dependencies    
+build_web_compilers  1.2.3     1.2.3       2.10.0      2.10.0    
+
+transitive dependencies
+_fe_analyzer_shared  -         -           2.0.0       2.0.0     
+analyzer             0.35.4    0.35.4      0.39.5      0.39.5    
+...
+
+9  dependencies are constrained to versions that are older than a resolvable version.
+To update these dependencies, edit pubspec.yaml.
+```
+
+As a rule, update the **dependencies** and **dev_dependencies** sections
+of your `pubspec.yaml` file
+to depend on the **resolvable** versions of out-of-date packages.
+
+Here are the meanings of the version numbers in the output:
+
+Current
+: The version used in your package, as recorded in `pubspec.lock`.
+  If the package isn't in `pubspec.lock`,
+  the value is `-`.
+
+Upgradable
+: The latest version allowed by your `pubspec.yaml` file.
+  This is the version that `pub upgrade` resolves to.
+  The value is `-` if the value in the **Current** column is `-`.
+
+Resolvable
+: The latest version that can be resolved,
+  when combined with all other dependencies.
+  This version corresponds to what you get if
+  you set all version constraints in `pubspec.yaml` to `any`
+  and then run `pub upgrade`.
+  A value of `-` means that the package won't be needed.
+
+{{site.alert.important}}
+  We don't recommend actually using `any` as the constraint,
+  because then running `pub upgrade` might surprise you by making
+  incompatible version changes.
+{{site.alert.end}}
+
+Latest
+: The latest version of the package available,
+  excluding prereleases unless you use the option `--prereleases`.
+
+For example, say your app depends on the `foo` and `bar` packages,
+but the latest version of `bar` depends on an old major version of `foo`.
+The result is that the latest _resolvable_ version of `foo`
+is different from the _latest_ version of `foo`.
+
+
+## Options
+
+For options that apply to all pub commands, see
+[Global options](/tools/pub/cmd#global-options).
+
+`--json`
+: Use this option to generate output in JSON format.
+
+`--[no-]color`
+: Use this option to change whether the output uses color for emphasis.
+  The default depends on whether you're using this command at a terminal.
+  At a terminal, `--color` is the default;
+  otherwise, `--no-color` is the default.
+
+`--[no-]up-to-date`
+: Use `--up-to-date` to make the output include dependencies that
+  are already at the latest version.
+  The default is `--no-up-to-date`, which saves space.
+
+`--[no-]prereleases`
+: Use `--prereleases` to include prereleases when determining
+  the latest package versions.
+  By default, prerelease versions aren't considered.
+
+`--[no-]dev-dependencies`
+: Use `--no-dev-dependencies` to ignore [dev dependencies][dev dependency].
+
+`--[no-]dependency-overrides`
+: Use `--no-dependency-overrides` to ignore package dependencies
```

Comment:
This is slightly off. The packages are not ignored, only the dependency_overrides on them are ignored.

Perhaps better would be:

`Use --no-dependency-overrides` to ignore [`dependency_overrides`][] when resolving package constraints.